jdi-light-angular-tests/src/main/java/io/github/com/pages/AngularPage.java

```diff
@@ -51,4 +52,52 @@
     public static Checkbox configurableResultCheckbox;
 
     public static SlideToggleSection slideToggleSection;
+
+    @UI("#mat-input-12")
+    public static TextField foodBasicInput;
+
+    @UI("#mat-input-13")
+    public static TextField leaveACommentBasicInput;
+
+    @UI("#mat-input-15")
+    public static TextField emailErrorStateMatcherInput;
+
+    @UI(".ng-tns-c94-39 mat-error")
+    public static Text errorStateMatcherMessageInput;
+
+    @UI("#mat-input-16")
+    public static TextArea autoSizeTextArea;
+
+    @UI("#mat-input-17")
+    public static TextArea clearableInput;
+
+    @UI(".ng-tns-c94-43 button")
+    public static Button clearableInputButton;
+
+    @UI("#mat-input-18")
+    public static TextField emailInput;
+
+    @UI(".ng-tns-c94-44 mat-error")
+    public static Text errorMessageInput;
+
+    @UI("input-form-example form")
+    public static InputsForm inputsForm;
+
+    @UI("#mat-input-27")
+    public static TextField messageHintInput;
+
+    @UI(".ng-tns-c94-53 mat-hint:first-of-type")
+    public static Text messageHint;
+
+    @UI(".ng-tns-c94-53 mat-hint:last-of-type")
+    public static Text messageCounterHint;
+
+    @UI(".ng-tns-c94-54 .mat-form-field-prefix")
+    public static UIElement prefixInput;
```

Comment:
If in jdi-light-html there is no suitable element, then please add this element to jdi-light-angular and use it intead of UIElement.